java/core/src/main/java/com/microsoft/bond/StructBondType.java

```diff
@@ -344,8 +357,18 @@ protected final TStruct deserializeValue(TaggedDeserializationContext context) t
         return value;
     }
 
+    @Override
+    protected final TStruct deserializeValue(UntaggedDeserializationContext context) throws IOException {
+        TStruct value = this.newDefaultValue();
+        if (this.baseStructType != null) {
+            this.baseStructType.deserializeValueAsBase(context, value);
+        }
+        this.deserializeStructFields(context, value);
```

Comment:
Talked offline - `deserializeValueAsBase(Untagged)` is now `deserializeValue(Untagged)` and fits nicely.